src/k8s-extension/HISTORY_private.rst

```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+0.4.0-beta.2

```

Comment:
Are these meant to be private changes or changes for public-preview?

Reply:
This change is meant to be for public preview, in that case should I just open a PR against the public branch or open PRs for both public and private branch?

Reply:
Also created a PR against the public branch. Thanks.
https://github.com/AzureArcForKubernetes/azure-cli-extensions/pull/42

Reply:
Just public, we handle downstream merging so when you are making a change that should be public, please make it against k8s-extension/public